Doxyfile

```diff
@@ -58,7 +58,7 @@ PROJECT_LOGO           =
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = ./docs
+OUTPUT_DIRECTORY       =
```

Comment:
What does having an empty output directory mean?

Reply:
If the output directory is empty the docs are produced at the starting location of doxygen

Reply:
This looks like it happened when I merged our existing Doxyfile.template into this root `Doxyfile` ... 

Because `HTML_OUTPUT = docs` this isn't a problem. However it might be a problem for folks who want to produce PDF docs or something from our repo.